tests/output_config_tests/expected_output_files/basic_connectivity_txt__stdout_output.txt

```diff
@@ -1,7 +1,6 @@
 global: cluster has 29 unique endpoints, 8 namespaces
 global: cluster has 29 unique endpoints, 8 namespaces
-
-final fw rules for query: :
+final fw rules for query: , config: output_config_tests/../example_policies/testcase8/testcase8-networkpolicy1.yaml:
```

Comment:
why this diff?

Reply:
When removing the "def _run_connectivity_map(self, configs_array, output_configuration)" from schemeRunner.py, I moved the following line :
`output_configuration.configName = config`
to the exec method of ConnectivityMapQuery:
       ` self.output_config.configName = self.config.name`
 (In order to keep printing the config name for connectivity map queries from scheme files as was done before the refactoring)
 As a result, the config name is printed now for connectivity map queries from both scheme and cmdline sources.